packages/meteor/debug.js

```diff
@@ -39,6 +39,11 @@ Meteor._debug = function (/* arguments */) {
 
         if (allArgumentsOfTypeString)
           console.log.apply(console, [Array.prototype.join.call(arguments, " ")]);
+        // We want to print out the stack trace when dynamics_nodejs.js throws
+        // an exception. Need to know what case this is and then make it into
+        // array so that we can pass it to console.log.apply.
+        else if (arguments.length === 2)
+          console.log.apply(console, [arguments[1]]);
```

Comment:
I don't think there's a particular reason to use `console.log.apply` and then immediately pass in an array literal. Looks like the line above this block code originated from a PR that wasn't fully following our style guide either.